### ISSUE TITLE
Add big ones

### ISSUE THREAD

**Issue by cookieguru** [jgrossi/elephpant.me#7]
I have three big ones that are currently looking for a new home

In addition, they are three different species, but I only have one of each.  The current model does not support putting your only copy up for trade.

**Comment by jgrossi** [comment]
yep, we already thought about the big ones and we're figuring out the best way to do that
about trading, the first idea is to trade only the ones you have double, for both sides.
we can think of some idea for selling some specific ones you want 💪

**Comment by wimg** [comment]
Adressed in #86
